packages/core/lib/model/map-context.ts

```diff
@@ -61,15 +61,10 @@ export interface MapContextLayerMapLibreStyle extends MapContextBaseLayer {
   accessToken?: string;
 }
 
-export interface MapContextLayerMvt extends MapContextBaseLayer {
-  type: "mvt";
-  url: string;
-  accessToken?: string;
-}
-
 export interface MapContextLayerXyz extends MapContextBaseLayer {
   type: "xyz";
   url: string;
+  tileFormat?: string;
```

Comment:
I think we should restrict the type here, and specifically rely on mime types; for MVT this would be `application/vnd.mapbox-vector-tile`.

So:
```typescript
tileFormat?: 'application/vnd.mapbox-vector-tile'
```

We should also clarify that not specifying any format will automatically fall back to assuming tiles are images